pkg/helm/values.go

```diff
@@ -0,0 +1,21 @@
+package helm
+
+import (
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type HelmValues map[string]interface{}
+
+func (h HelmValues) GetString(key string) (string, bool, error) {
```

Comment:
Can we have comments for those exported functions?

Reply:
Done.